[2023-08-24T09:28:18.377+0000] {processor.py:157} INFO - Started process (PID=37) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:28:18.482+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T09:28:18.534+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:28:18.508+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:28:18.874+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:28:18.820+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T09:28:18.912+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:28:19.423+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 1.158 seconds
[2023-08-24T09:30:27.030+0000] {processor.py:157} INFO - Started process (PID=113) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:30:27.147+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T09:30:27.177+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:30:27.174+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:30:27.319+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:30:27.281+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T09:30:27.324+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:30:27.629+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.700 seconds
[2023-08-24T09:31:08.887+0000] {processor.py:157} INFO - Started process (PID=158) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:31:08.898+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T09:31:08.909+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:31:08.907+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:31:09.021+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:31:08.961+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T09:31:09.040+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:31:10.023+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 1.165 seconds
[2023-08-24T09:31:48.862+0000] {processor.py:157} INFO - Started process (PID=214) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:31:48.883+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T09:31:48.911+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:31:48.908+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:31:49.048+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:31:49.019+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T09:31:49.057+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:31:50.604+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 1.925 seconds
[2023-08-24T09:32:53.477+0000] {processor.py:157} INFO - Started process (PID=262) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:32:53.484+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T09:32:53.494+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:32:53.493+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:32:53.583+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:32:53.565+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T09:32:53.594+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:32:53.728+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.264 seconds
[2023-08-24T09:33:49.341+0000] {processor.py:157} INFO - Started process (PID=320) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:33:49.349+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T09:33:49.371+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:33:49.370+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:33:49.409+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:33:49.398+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T09:33:49.413+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:33:49.513+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.186 seconds
[2023-08-24T09:34:20.118+0000] {processor.py:157} INFO - Started process (PID=374) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:34:20.121+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T09:34:20.124+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:34:20.124+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:34:20.139+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:34:20.136+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T09:34:20.141+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:34:20.167+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.055 seconds
[2023-08-24T09:34:50.590+0000] {processor.py:157} INFO - Started process (PID=429) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:34:50.592+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T09:34:50.596+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:34:50.596+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:34:50.612+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:34:50.608+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T09:34:50.614+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:34:50.640+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.055 seconds
[2023-08-24T09:35:20.950+0000] {processor.py:157} INFO - Started process (PID=485) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:35:20.954+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T09:35:20.966+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:35:20.964+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:35:20.991+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:35:20.986+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T09:35:21.000+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:35:21.038+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.095 seconds
[2023-08-24T09:35:51.743+0000] {processor.py:157} INFO - Started process (PID=541) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:35:51.746+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T09:35:51.750+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:35:51.749+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:35:51.768+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:35:51.762+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T09:35:51.770+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:35:51.799+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.064 seconds
[2023-08-24T09:36:21.953+0000] {processor.py:157} INFO - Started process (PID=597) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:36:21.956+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T09:36:21.962+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:36:21.962+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:36:21.975+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:36:21.972+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T09:36:21.976+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:36:22.009+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.063 seconds
[2023-08-24T09:36:52.124+0000] {processor.py:157} INFO - Started process (PID=652) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:36:52.128+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T09:36:52.135+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:36:52.134+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:36:52.151+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:36:52.148+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T09:36:52.152+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:36:52.185+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.066 seconds
[2023-08-24T09:38:04.618+0000] {processor.py:157} INFO - Started process (PID=31) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:38:04.625+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T09:38:04.636+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:38:04.635+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:38:04.677+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:38:04.669+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T09:38:04.683+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:38:04.787+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.180 seconds
[2023-08-24T09:38:34.881+0000] {processor.py:157} INFO - Started process (PID=40) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:38:34.883+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T09:38:34.888+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:38:34.887+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:38:34.904+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:38:34.899+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T09:38:34.906+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:38:34.941+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.066 seconds
[2023-08-24T09:39:05.535+0000] {processor.py:157} INFO - Started process (PID=49) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:39:05.542+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T09:39:05.563+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:39:05.561+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:39:05.619+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:39:05.604+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T09:39:05.632+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:39:05.717+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.209 seconds
[2023-08-24T09:39:36.156+0000] {processor.py:157} INFO - Started process (PID=58) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:39:36.160+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T09:39:36.166+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:39:36.165+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:39:36.186+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:39:36.179+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T09:39:36.188+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:39:36.243+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.093 seconds
[2023-08-24T09:40:06.720+0000] {processor.py:157} INFO - Started process (PID=67) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:40:06.724+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T09:40:06.732+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:40:06.731+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:40:06.753+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:40:06.748+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T09:40:06.755+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:40:06.804+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.093 seconds
[2023-08-24T09:40:37.388+0000] {processor.py:157} INFO - Started process (PID=77) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:40:37.391+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T09:40:37.400+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:40:37.399+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:40:37.421+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:40:37.416+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T09:40:37.424+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:40:37.466+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.095 seconds
[2023-08-24T09:41:07.874+0000] {processor.py:157} INFO - Started process (PID=86) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:41:07.876+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T09:41:07.879+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:41:07.879+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:41:07.892+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:41:07.890+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T09:41:07.894+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:41:07.918+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.048 seconds
[2023-08-24T09:41:38.295+0000] {processor.py:157} INFO - Started process (PID=95) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:41:38.297+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T09:41:38.302+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:41:38.302+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:41:38.316+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:41:38.312+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T09:41:38.318+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:41:38.345+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.056 seconds
[2023-08-24T09:42:08.699+0000] {processor.py:157} INFO - Started process (PID=115) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:42:08.702+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T09:42:08.707+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:42:08.706+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:42:08.723+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:42:08.719+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T09:42:08.727+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:42:08.756+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.064 seconds
[2023-08-24T09:42:39.413+0000] {processor.py:157} INFO - Started process (PID=124) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:42:39.420+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T09:42:39.434+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:42:39.431+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:42:39.483+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:42:39.475+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T09:42:39.487+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:42:39.545+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.151 seconds
[2023-08-24T09:43:10.070+0000] {processor.py:157} INFO - Started process (PID=133) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:43:10.073+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T09:43:10.080+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:43:10.079+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:43:10.096+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:43:10.092+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T09:43:10.097+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:43:10.147+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.083 seconds
[2023-08-24T09:43:40.546+0000] {processor.py:157} INFO - Started process (PID=142) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:43:40.552+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T09:43:40.559+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:43:40.558+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:43:40.578+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:43:40.573+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T09:43:40.581+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:43:40.625+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.087 seconds
[2023-08-24T09:44:11.110+0000] {processor.py:157} INFO - Started process (PID=151) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:44:11.113+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T09:44:11.121+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:44:11.120+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:44:11.156+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:44:11.149+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T09:44:11.162+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:44:11.214+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.115 seconds
[2023-08-24T09:44:41.561+0000] {processor.py:157} INFO - Started process (PID=160) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:44:41.564+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T09:44:41.569+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:44:41.568+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:44:41.585+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:44:41.581+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T09:44:41.587+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:44:41.634+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.079 seconds
[2023-08-24T09:45:11.986+0000] {processor.py:157} INFO - Started process (PID=169) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:45:11.989+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T09:45:11.995+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:45:11.994+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:45:12.010+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:45:12.007+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T09:45:12.014+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:45:12.059+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.080 seconds
[2023-08-24T09:45:42.444+0000] {processor.py:157} INFO - Started process (PID=178) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:45:42.447+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T09:45:42.452+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:45:42.452+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:45:42.467+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:45:42.463+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T09:45:42.469+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:45:42.499+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.063 seconds
[2023-08-24T09:46:12.810+0000] {processor.py:157} INFO - Started process (PID=187) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:46:12.813+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T09:46:12.826+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:46:12.825+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:46:12.848+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:46:12.841+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T09:46:12.851+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:46:12.901+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.099 seconds
[2023-08-24T09:46:43.376+0000] {processor.py:157} INFO - Started process (PID=196) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:46:43.381+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T09:46:43.387+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:46:43.386+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:46:43.403+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:46:43.399+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T09:46:43.407+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:46:43.440+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.079 seconds
[2023-08-24T09:47:13.968+0000] {processor.py:157} INFO - Started process (PID=206) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:47:13.973+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T09:47:13.983+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:47:13.982+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:47:14.010+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:47:14.005+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T09:47:14.012+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:47:14.067+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.108 seconds
[2023-08-24T09:47:44.672+0000] {processor.py:157} INFO - Started process (PID=216) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:47:44.677+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T09:47:44.687+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:47:44.686+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:47:44.721+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:47:44.714+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T09:47:44.724+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:47:44.776+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.134 seconds
[2023-08-24T09:48:15.418+0000] {processor.py:157} INFO - Started process (PID=225) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:48:15.422+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T09:48:15.430+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:48:15.429+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:48:15.453+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:48:15.446+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T09:48:15.459+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:48:15.500+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.092 seconds
[2023-08-24T09:48:45.897+0000] {processor.py:157} INFO - Started process (PID=235) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:48:45.901+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T09:48:45.907+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:48:45.906+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:48:45.921+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:48:45.917+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T09:48:45.924+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:48:45.954+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.063 seconds
[2023-08-24T09:49:16.342+0000] {processor.py:157} INFO - Started process (PID=244) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:49:16.346+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T09:49:16.351+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:49:16.351+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:49:16.367+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:49:16.364+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T09:49:16.369+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:49:16.399+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.062 seconds
[2023-08-24T09:49:46.736+0000] {processor.py:157} INFO - Started process (PID=253) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:49:46.739+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T09:49:46.745+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:49:46.744+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:49:46.761+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:49:46.757+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T09:49:46.765+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:49:46.798+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.070 seconds
[2023-08-24T09:50:17.162+0000] {processor.py:157} INFO - Started process (PID=262) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:50:17.165+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T09:50:17.170+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:50:17.169+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:50:17.183+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:50:17.180+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T09:50:17.185+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:50:17.222+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.064 seconds
[2023-08-24T09:50:47.558+0000] {processor.py:157} INFO - Started process (PID=271) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:50:47.561+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T09:50:47.565+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:50:47.565+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:50:47.580+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:50:47.577+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T09:50:47.582+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:50:47.609+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.058 seconds
[2023-08-24T09:51:18.001+0000] {processor.py:157} INFO - Started process (PID=280) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:51:18.003+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T09:51:18.008+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:51:18.008+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:51:18.022+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:51:18.019+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T09:51:18.026+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:51:18.052+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.056 seconds
[2023-08-24T09:51:48.433+0000] {processor.py:157} INFO - Started process (PID=289) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:51:48.435+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T09:51:48.443+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:51:48.442+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:51:48.462+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:51:48.459+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T09:51:48.463+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:51:48.493+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.066 seconds
[2023-08-24T09:52:18.813+0000] {processor.py:157} INFO - Started process (PID=298) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:52:18.816+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T09:52:18.821+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:52:18.820+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:52:18.833+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:52:18.830+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T09:52:18.836+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:52:18.865+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.057 seconds
[2023-08-24T09:52:49.224+0000] {processor.py:157} INFO - Started process (PID=306) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:52:49.227+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T09:52:49.234+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:52:49.233+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:52:49.250+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:52:49.246+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T09:52:49.253+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:52:49.296+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.079 seconds
[2023-08-24T09:53:19.721+0000] {processor.py:157} INFO - Started process (PID=315) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:53:19.724+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T09:53:19.730+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:53:19.729+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:53:19.746+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:53:19.743+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T09:53:19.749+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:53:19.791+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.077 seconds
[2023-08-24T09:53:50.236+0000] {processor.py:157} INFO - Started process (PID=325) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:53:50.241+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T09:53:50.249+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:53:50.248+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:53:50.271+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:53:50.262+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T09:53:50.275+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:53:50.327+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.097 seconds
[2023-08-24T09:54:20.736+0000] {processor.py:157} INFO - Started process (PID=334) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:54:20.739+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T09:54:20.744+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:54:20.743+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:54:20.767+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:54:20.761+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T09:54:20.772+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:54:20.813+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.084 seconds
[2023-08-24T09:54:51.173+0000] {processor.py:157} INFO - Started process (PID=343) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:54:51.177+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T09:54:51.184+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:54:51.183+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:54:51.207+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:54:51.201+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T09:54:51.209+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:54:51.242+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.075 seconds
[2023-08-24T09:55:21.615+0000] {processor.py:157} INFO - Started process (PID=351) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:55:21.620+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T09:55:21.627+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:55:21.626+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:55:21.643+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:55:21.640+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T09:55:21.645+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:55:21.678+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.069 seconds
[2023-08-24T09:55:52.058+0000] {processor.py:157} INFO - Started process (PID=360) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:55:52.060+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T09:55:52.067+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:55:52.066+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:55:52.104+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:55:52.098+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T09:55:52.106+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:55:52.151+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.100 seconds
[2023-08-24T09:56:22.581+0000] {processor.py:157} INFO - Started process (PID=369) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:56:22.583+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T09:56:22.589+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:56:22.588+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:56:22.604+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:56:22.600+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T09:56:22.607+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:56:22.638+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.062 seconds
[2023-08-24T09:56:52.984+0000] {processor.py:157} INFO - Started process (PID=378) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:56:52.988+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T09:56:53.002+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:56:53.001+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:56:53.021+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:56:53.015+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T09:56:53.023+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:56:53.061+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.087 seconds
[2023-08-24T09:57:23.471+0000] {processor.py:157} INFO - Started process (PID=387) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:57:23.473+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T09:57:23.478+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:57:23.478+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:57:23.497+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:57:23.493+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T09:57:23.500+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:57:23.537+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.071 seconds
[2023-08-24T09:57:53.868+0000] {processor.py:157} INFO - Started process (PID=397) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:57:53.871+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T09:57:53.877+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:57:53.876+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:57:53.897+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:57:53.892+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T09:57:53.899+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:57:53.932+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.073 seconds
[2023-08-24T09:58:24.342+0000] {processor.py:157} INFO - Started process (PID=407) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:58:24.345+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T09:58:24.350+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:58:24.349+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:58:24.374+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:58:24.370+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T09:58:24.376+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:58:24.412+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.076 seconds
[2023-08-24T09:58:55.067+0000] {processor.py:157} INFO - Started process (PID=417) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:58:55.070+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T09:58:55.078+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:58:55.077+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:58:55.096+0000] {logging_mixin.py:151} INFO - [2023-08-24T09:58:55.091+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T09:58:55.098+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T09:58:55.153+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.094 seconds
[2023-08-24T10:00:02.776+0000] {processor.py:157} INFO - Started process (PID=31) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:00:02.782+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T10:00:02.792+0000] {logging_mixin.py:151} INFO - [2023-08-24T10:00:02.791+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:00:02.826+0000] {logging_mixin.py:151} INFO - [2023-08-24T10:00:02.815+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T10:00:02.830+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:00:02.918+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.151 seconds
[2023-08-24T10:00:33.875+0000] {processor.py:157} INFO - Started process (PID=41) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:00:33.878+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T10:00:33.889+0000] {logging_mixin.py:151} INFO - [2023-08-24T10:00:33.887+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:00:33.922+0000] {logging_mixin.py:151} INFO - [2023-08-24T10:00:33.914+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T10:00:33.926+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:00:33.992+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.124 seconds
[2023-08-24T10:01:04.965+0000] {processor.py:157} INFO - Started process (PID=55) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:01:04.975+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T10:01:05.004+0000] {logging_mixin.py:151} INFO - [2023-08-24T10:01:05.003+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:01:05.100+0000] {logging_mixin.py:151} INFO - [2023-08-24T10:01:05.062+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T10:01:05.108+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:01:05.275+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.354 seconds
[2023-08-24T10:01:36.266+0000] {processor.py:157} INFO - Started process (PID=69) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:01:36.269+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T10:01:36.275+0000] {logging_mixin.py:151} INFO - [2023-08-24T10:01:36.274+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:01:36.303+0000] {logging_mixin.py:151} INFO - [2023-08-24T10:01:36.299+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T10:01:36.305+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:01:36.341+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.081 seconds
[2023-08-24T10:02:06.706+0000] {processor.py:157} INFO - Started process (PID=79) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:02:06.714+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T10:02:06.734+0000] {logging_mixin.py:151} INFO - [2023-08-24T10:02:06.733+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:02:06.756+0000] {logging_mixin.py:151} INFO - [2023-08-24T10:02:06.751+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T10:02:06.759+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:02:06.803+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.104 seconds
[2023-08-24T10:02:37.409+0000] {processor.py:157} INFO - Started process (PID=89) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:02:37.413+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T10:02:37.422+0000] {logging_mixin.py:151} INFO - [2023-08-24T10:02:37.421+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:02:37.435+0000] {logging_mixin.py:151} INFO - [2023-08-24T10:02:37.432+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T10:02:37.437+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:02:37.476+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.075 seconds
[2023-08-24T10:03:07.888+0000] {processor.py:157} INFO - Started process (PID=99) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:03:07.891+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T10:03:07.895+0000] {logging_mixin.py:151} INFO - [2023-08-24T10:03:07.895+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:03:07.909+0000] {logging_mixin.py:151} INFO - [2023-08-24T10:03:07.906+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T10:03:07.911+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:03:07.940+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.059 seconds
[2023-08-24T10:03:38.325+0000] {processor.py:157} INFO - Started process (PID=110) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:03:38.331+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T10:03:38.337+0000] {logging_mixin.py:151} INFO - [2023-08-24T10:03:38.337+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:03:38.357+0000] {logging_mixin.py:151} INFO - [2023-08-24T10:03:38.352+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T10:03:38.359+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:03:38.414+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.100 seconds
[2023-08-24T10:04:08.892+0000] {processor.py:157} INFO - Started process (PID=120) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:04:08.898+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T10:04:08.927+0000] {logging_mixin.py:151} INFO - [2023-08-24T10:04:08.926+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:04:08.952+0000] {logging_mixin.py:151} INFO - [2023-08-24T10:04:08.942+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T10:04:08.956+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:04:09.013+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.136 seconds
[2023-08-24T10:04:39.533+0000] {processor.py:157} INFO - Started process (PID=130) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:04:39.538+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T10:04:39.547+0000] {logging_mixin.py:151} INFO - [2023-08-24T10:04:39.546+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:04:39.569+0000] {logging_mixin.py:151} INFO - [2023-08-24T10:04:39.562+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T10:04:39.572+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:04:39.617+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.095 seconds
[2023-08-24T10:05:10.018+0000] {processor.py:157} INFO - Started process (PID=140) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:05:10.022+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T10:05:10.037+0000] {logging_mixin.py:151} INFO - [2023-08-24T10:05:10.037+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:05:10.058+0000] {logging_mixin.py:151} INFO - [2023-08-24T10:05:10.053+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T10:05:10.060+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:05:10.110+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.098 seconds
[2023-08-24T10:05:40.647+0000] {processor.py:157} INFO - Started process (PID=150) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:05:40.653+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T10:05:40.659+0000] {logging_mixin.py:151} INFO - [2023-08-24T10:05:40.659+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:05:40.702+0000] {logging_mixin.py:151} INFO - [2023-08-24T10:05:40.689+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T10:05:40.706+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:05:40.763+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.126 seconds
[2023-08-24T10:06:11.260+0000] {processor.py:157} INFO - Started process (PID=160) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:06:11.265+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T10:06:11.275+0000] {logging_mixin.py:151} INFO - [2023-08-24T10:06:11.273+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:06:11.295+0000] {logging_mixin.py:151} INFO - [2023-08-24T10:06:11.291+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T10:06:11.297+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:06:11.337+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.083 seconds
[2023-08-24T10:06:41.869+0000] {processor.py:157} INFO - Started process (PID=170) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:06:41.874+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T10:06:41.882+0000] {logging_mixin.py:151} INFO - [2023-08-24T10:06:41.881+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:06:41.899+0000] {logging_mixin.py:151} INFO - [2023-08-24T10:06:41.894+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T10:06:41.901+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:06:41.941+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.079 seconds
[2023-08-24T10:07:12.432+0000] {processor.py:157} INFO - Started process (PID=180) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:07:12.438+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T10:07:12.445+0000] {logging_mixin.py:151} INFO - [2023-08-24T10:07:12.445+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:07:12.464+0000] {logging_mixin.py:151} INFO - [2023-08-24T10:07:12.459+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T10:07:12.467+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:07:12.520+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.095 seconds
[2023-08-24T10:07:43.069+0000] {processor.py:157} INFO - Started process (PID=189) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:07:43.073+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T10:07:43.080+0000] {logging_mixin.py:151} INFO - [2023-08-24T10:07:43.079+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:07:43.096+0000] {logging_mixin.py:151} INFO - [2023-08-24T10:07:43.092+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T10:07:43.098+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:07:43.135+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.073 seconds
[2023-08-24T10:08:13.535+0000] {processor.py:157} INFO - Started process (PID=199) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:08:13.538+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T10:08:13.543+0000] {logging_mixin.py:151} INFO - [2023-08-24T10:08:13.542+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:08:13.561+0000] {logging_mixin.py:151} INFO - [2023-08-24T10:08:13.556+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T10:08:13.563+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:08:13.606+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.076 seconds
[2023-08-24T10:08:43.955+0000] {processor.py:157} INFO - Started process (PID=209) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:08:43.959+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T10:08:43.968+0000] {logging_mixin.py:151} INFO - [2023-08-24T10:08:43.967+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:08:43.992+0000] {logging_mixin.py:151} INFO - [2023-08-24T10:08:43.988+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T10:08:43.994+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:08:44.035+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.092 seconds
[2023-08-24T10:09:14.516+0000] {processor.py:157} INFO - Started process (PID=219) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:09:14.520+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T10:09:14.529+0000] {logging_mixin.py:151} INFO - [2023-08-24T10:09:14.528+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:09:14.548+0000] {logging_mixin.py:151} INFO - [2023-08-24T10:09:14.543+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T10:09:14.550+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:09:14.594+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.095 seconds
[2023-08-24T10:43:56.967+0000] {processor.py:157} INFO - Started process (PID=230) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:43:56.975+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T10:43:56.987+0000] {logging_mixin.py:151} INFO - [2023-08-24T10:43:56.986+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:43:57.007+0000] {logging_mixin.py:151} INFO - [2023-08-24T10:43:57.002+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T10:43:57.009+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T10:43:57.069+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.108 seconds
[2023-08-24T12:56:49.618+0000] {processor.py:157} INFO - Started process (PID=240) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T12:56:49.625+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T12:56:49.645+0000] {logging_mixin.py:151} INFO - [2023-08-24T12:56:49.644+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T12:56:49.679+0000] {logging_mixin.py:151} INFO - [2023-08-24T12:56:49.669+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T12:56:49.682+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T12:56:49.755+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.144 seconds
[2023-08-24T12:59:27.432+0000] {processor.py:157} INFO - Started process (PID=250) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T12:59:27.435+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T12:59:27.441+0000] {logging_mixin.py:151} INFO - [2023-08-24T12:59:27.441+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T12:59:27.467+0000] {logging_mixin.py:151} INFO - [2023-08-24T12:59:27.462+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T12:59:27.470+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T12:59:27.517+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.093 seconds
[2023-08-24T14:07:13.691+0000] {processor.py:157} INFO - Started process (PID=260) to work on /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T14:07:13.697+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2023-08-24T14:07:13.704+0000] {logging_mixin.py:151} INFO - [2023-08-24T14:07:13.704+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T14:07:13.726+0000] {logging_mixin.py:151} INFO - [2023-08-24T14:07:13.720+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2023-08-24T14:07:13.728+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2023-08-24T14:07:13.791+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.109 seconds
